changelogs/fragments/811-aws_s3-ovewrite-default.yml

```diff
@@ -0,0 +1,2 @@
+breaking_changes:
+- aws_s3 - ensure overwrite=different instead of always to have the module idempotent by default (https://github.com/ansible-collections/amazon.aws/issues/811).
```

Comment:
```suggestion
- aws_s3 - the default value for ``ensure overwrite`` has been changed to ``different`` instead of ``always`` so that the module is idempotent by default (https://github.com/ansible-collections/amazon.aws/issues/811).
```